KsApi/models/Backing.swift

```diff
@@ -21,6 +21,29 @@ public struct Backing {
   public let shippingAmount: Int?
   public let status: Status
 
+  public struct PaymentSource {
+    public let expirationDate: String?
+    public let id: String?
+    public let lastFour: String?
+    public let paymentType: PaymentType?
```

Comment:
I think this should be non-optional.

Reply:
I was following the description of this [ticket](https://dripsprint.atlassian.net/secure/RapidBoard.jspa?rapidView=9&modal=detail&selectedIssue=NT-422&assignee=5d41b601f14e7c0da562b03f)

Reply:
I think the ticket says it should be non-optional? 🤔 
![image](https://user-images.githubusercontent.com/3156796/66962248-84e8d200-f03e-11e9-837d-ad506b6f8816.png)

Reply:
OMG! paymentType is non-optional, but type is! Got lost between types 😅